ci-operator/step-registry/sandboxed-containers-operator/create-prowjob/sandboxed-containers-operator-create-prowjob-commands.sh

```diff
@@ -370,8 +370,8 @@ fi
 
 # Basic YAML syntax validation if yq is available
 if command -v yq &> /dev/null; then
-    echo "Validating YAML syntax with yq..."
-    if yq eval '.' "${OUTPUT_FILE}" >/dev/null 2>&1; then
+    echo "Validating YAML: yq eval '.' "${OUTPUT_FILE}"
+    if yq eval '.' "${OUTPUT_FILE}" ; then
```

Comment:
Since it's not sent automatically and the `$OUTPUT_FILE` is listed in the list of steps I don't think it should be printed here (maybe we could change the `cat ${OUTPUT_FILE}"` for `yq ...`, though)